test/suite/test_prepare04.py

```diff
@@ -96,8 +96,8 @@ def test_prepare_conflict(self):
         c.set_value(2)
         c.update()
         self.session.prepare_transaction('prepare_timestamp=' + self.prepare_ts)
-        conflictmsg = '/conflict between concurrent operations/'
-        preparemsg = '/conflict with a prepared update/'
+        conflictmsg = "WT_ROLLBACK: conflict between concurrent operations"
+        preparemsg = "WT_PREPARE_CONFLICT: conflict with a prepared update"
```

Comment:
I think you don't need the change above. The form with `/string/` will match on that as a substring of the message so I don't think you need to make the message that much more detailed. It should be fine as-is.

Reply:
@sueloverso Thanks, changed the error strings as original substrings